params/config.go

```diff
@@ -263,6 +257,42 @@ func (c *ChainConfig) Description() string {
 	return banner
 }
 
+func (c *ChainConfig) SetNetworkUpgradeDefaults() {
```

Comment:
This will remove the capability of giving `nil` values for upgrades/forks to disable them. I don't think the current codebase supports deactivating any of those, but if we want to maintain the "full" backward-compatibility we can expand the `*uint64/*bigInt` types to some struct with which we can deduct information about `nil` vs `value` vs `blank` in json files. This would require a bigger change in the code and would increase the diff between the upstream.